ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/console/CPPOutputCustomizer.java

```diff
@@ -0,0 +1,184 @@
+/*******************************************************************************
```

Comment:
In theory yes, but... to let `CPPOutputCustomizer` do its job I have to create its instance and add it to `CompoundOutputCustomizer` that is created in `CommandOutputConsolePresenter`, as well as in `DefaultOutputConsole`, in `che-core-ide-app` that doesn't have a dependency to `plugin-cpp`.

How we can deal with it? Move the class to `che-plugin-cpp-lang-shared` and make `che-core-ide-app` to depend on it?

But I feel like it would be better to move `*OutputCustomizer`-s to according plug-ins and make it possible to gather the required customizers on base of workspace's activated stacks/runtimes or a workspace-related configuration preference.